padle/padle.py

```diff
@@ -0,0 +1,537 @@
+            embed_thumbnail=EmbedThumbnail(
+                MonsterImage.icon(guess_monster.monster_id))).to_embed()
+        confirmation = await self.get_embed_user_conf(ctx, m_embed, timeout=20)
+        if confirmation is None:
+            return await send_cancellation_message(ctx, "Confirmation timeout.")
+        if confirmation is False:
+            return await send_cancellation_message(ctx, "Please guess again.")
+        if not await self.config.user(ctx.author).start():
+            return await send_cancellation_message(ctx, "You have not started a game yet! The PADle may have just expired.")
+        if await self.config.user(ctx.author).done():
+            return await send_cancellation_message(ctx, "You have already finished today's PADle!")
+            
+        points = 0
+        async with self.config.user(ctx.author).guesses() as guesses:
+            guesses.append(guess_monster.monster_id)
+        monster = await dbcog.find_monster(str(await self.config.padle_today()), ctx.author.id)
+        line = await self.get_name_line(ctx, monster, guess_monster)
+        other_line = await self.get_other_info_line(ctx, monster, guess_monster)
+        awakenings_line = await self.get_awakenings_line(ctx, monster, guess_monster)
+        data = [line[0], other_line[0], awakenings_line[0]]
+        points += line[1] + other_line[1] + awakenings_line[1]
+
+        async with self.config.user(ctx.author).embed_text() as descr:
+            descr.append("\n".join(data))
+        embed_texts = await self.config.user(ctx.author).embed_text()
+        channel = self.bot.get_channel(await self.config.user(ctx.author).channel_id())
+        del_msg = await channel.fetch_message(await self.config.user(ctx.author).edit_id())
+        await del_msg.delete()
+        embed_pages = []
+        embed_controls = {
+            "\N{LEFTWARDS BLACK ARROW}\N{VARIATION SELECTOR-16}": prev_page,
+            "\N{BLACK RIGHTWARDS ARROW}\N{VARIATION SELECTOR-16}": next_page,
+        }
+        emojis = ["\N{LEFTWARDS BLACK ARROW}\N{VARIATION SELECTOR-16}",
+                  "\N{BLACK RIGHTWARDS ARROW}\N{VARIATION SELECTOR-16}"]
+        num_pages = ceil(len(embed_texts) / 5)
+        for pageNum in range(ceil(len(embed_texts) / 5)):
+            embed = discord.Embed(title="Padle #{}".format(await self.config.num_days()), type="rich")
+            for index, text in enumerate(embed_texts[:5]):
+                embed.add_field(name="**Guess {}**:".format(index + 1 + pageNum * 5), value=text, inline=False)
+            embed.set_footer(text="Page {}/{}".format((pageNum + 1), num_pages))
+            embed_texts = embed_texts[5:]
+            embed_pages.append(embed)
+        message = await ctx.send(embed=discord.Embed(title="Padle"))
+        await self.config.user(ctx.author).edit_id.set(message.id)
+        await self.config.user(ctx.author).channel_id.set(ctx.channel.id)
+        score = ":yellow_square:"
```

Comment:
It's preferable to use the actual unicode character over the discord shortcut.  You can find what it is via `[p]unicodename <glyph>` with the emoji cog loaded ([p] is your bot's prefix).  The unicode escape for this glyph is `\N{LARGE YELLOW SQUARE}`